src/Core/Components/Checkbox/FluentCheckbox.razor

```diff
@@ -0,0 +1,39 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentInputImmediateBase<bool?>
```

Comment:
I don't think you need to use the `Immemdiate`

Reply:
To be honest, I've just duplicate the FluentInputText and start the work from these. So sure I will have no used code. thank you

Reply:
Always better to start from an empty class and copy only required items (not the inverse) 😀

Reply:
Too lazy :D